src/Cuboid.cpp

```diff
@@ -24,10 +24,10 @@ static bool DoIntervalsIntersect(int a_Min1, int a_Max1, int a_Min2, int a_Max2)
 ////////////////////////////////////////////////////////////////////////////////
 // cCuboid:
 
-cCuboid & cCuboid::operator =(cCuboid a_Other)
+cCuboid & cCuboid::operator =(const cCuboid & a_Other)
```

Comment:
I can't see why it's even defined in the first place, it's nothing the compiler wouldn't generate anyway.